### ISSUE TITLE
Severity classifier NullPointerException

### ISSUE THREAD

**Issue by jrwilliams** [ossmeter/ossmeter#34]
I've encountered a NullPointerException for the severity classifier. The line in question is actually a System.out.println(...) statement, but I assume that if the print statement wasn't there it would also have a knock-on effect.

Project: AMW - Atlas Model Weaver

Date: 20100325

To reproduce you can import the project as follows:

``` java
Platform platform = ...
EclipseProjectImporter im = new EclipseProjectImporter();
im.importProjectByUrl("https://projects.eclipse.org/projects/modeling.gmt.amw", platform);
```

The stack trace:

``` java
ERROR [MetricListExecutor (modeling-gmt-amw, 20100325)] (03:01:49): Exception thrown during metric provider execution (severityclassification).
java.lang.NullPointerException
        at org.ossmeter.severityclassifier.opennlptartarus.libsvm.ClassificationInstance.initialiseMetadata(ClassificationInstance.java:104)
        at org.ossmeter.severityclassifier.opennlptartarus.libsvm.ClassificationInstance.<init>(ClassificationInstance.java:65)
        at org.ossmeter.severityclassifier.opennlptartarus.libsvm.ClassificationInstanceCollection.add(ClassificationInstanceCollection.java:56)
        at org.ossmeter.severityclassifier.opennlptartarus.libsvm.Classifier.add(Classifier.java:54)
        at org.ossmeter.metricprovider.trans.severityclassification.SeverityClassificationTransMetricProvider.measure(SeverityClassificationTransMetricProvider.java:195)
        at org.ossmeter.metricprovider.trans.severityclassification.SeverityClassificationTransMetricProvider.measure(SeverityClassificationTransMetricProvider.java:1)
        at org.ossmeter.platform.osgi.executors.MetricListExecutor.run(MetricListExecutor.java:130)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:744

```

**Comment by jrwilliams** [comment]
This has also occurred for the Epsilon project (date 20100325) - I'm guessing this might be happening on the first day that a message is posted?

**Comment by yannisKork** [comment]
Is this exception encountered on the NNTP channel or the Bugzilla channel of this project?

**Comment by yannisKork** [comment]
Most probably, the bug refers to the NNTP channel. The Bugzilla channel has been processed correctly.

**Comment by yannisKork** [comment]
I was not able to reproduce the errors.

**Comment by jrwilliams** [comment]
Reopening this as it has also occurred for the following projects:
- modeling-graphiti, 20150220
- modeling-mmt-atl, 20150212
- modeling-emft-refactor, 20141030
- modeling-mdt-xsd, 20150205

I do wonder whether it is something that has gone wrong at our end, as it does seem intermittent. I'll try to reproduce these myself today

**Comment by yannisKork** [comment]
I suspect that sometimes some message metadata are not being downloaded fully. I have not managed to reproduce any of these error for specific dates. However, every now and then I have encountered errors such as this one or the error in issue 35. Even in these cases, the errors cannot be reproduce; the problematic dates process fine on rerun.

**Comment by jrwilliams** [comment]
Okey dokey, thanks Yannis. Let's leave this issue open in case other people encounter it.

**Comment by kolovos** [comment]
Fixed in commit 7c71a4031e65de92d26788cd6c01ff517ffc84c8
